lib/data/config.py

```diff
@@ -0,0 +1,9 @@
+param = {
```

Comment:
Please see the sample config format of [IRCLogParser](https://github.com/prasadtalasila/IRCLogParser/blob/master/lib/config.py)

Reply:
The right location for the config file is the top-level of the git repo.